doc/sphinx/user-extensions/syntax-extensions.rst

```diff
@@ -1247,23 +1248,24 @@ Here are the syntax elements used by the various notation commands.
              due to legacy notation in the Coq standard library.
              It can be turned on with the ``-w disj-pattern-notation`` flag.
 
+.. note::
+
+   The entry ``strict ident`` parses identifiers as in grammar entry
+   :n:`@ident`. The entry ``name`` parses :n:`@name`, that is
+   identifiers and :g:`_`. The entry ``global`` parses :n:`@qualid`
+   (and thus, as a special case, :n:`@ident`). The entry ``bigint``
+   parses arbitrary large integer numbers as in :n:`@bigint`.
+
 .. note::
 
    As of version 8.13, the entry ``ident`` is a deprecated alias for
-   ``name``. In the future, it is planned to strictly parse an
-   identifier (to the exclusion of :g:`_`). If the intent was to use
-   ``ident`` as an identifier (excluding :g:`_`), just silence the warning with
-   :n:`Set Warnings "-deprecated-ident-entry"` and it should automatically
-   get its intended meaning in version 8.15.
-
-   Similarly, ``as ident`` is a deprecated alias for ``as name``, which
-   will only accept an identifier in the future.  If the
-   intent was to use ``as ident`` as an identifier
-   (excluding :g:`_`), just silence the warning with
-   :n:`Set Warnings "-deprecated-as-ident-kind"`.
-
-   However, this deprecation does not apply to custom entries, where it
-   already denotes an identifier, as expected.
+   ``name``. If the intent is to use ``ident`` as an identifier
+   (excluding :g:`_`), use ``strict ident`` instead.  In the future,
+   ``ident`` may be rebound to :n:`@ident`.  However, this does not
+   apply to custom entries, where ``ident`` already parses
+   :n:`@ident`. This does not apply either to binders parsed as
+   :n:`@term` but interpreted as identifiers where the ``as ident``
+   modifier already excludes :g:`_`.
```

Comment:
```suggestion
   :n:`@ident`. Nor does it apply to binders parsed as
   :n:`@term` but interpreted as identifiers where the ``as ident``
   modifier already excludes :g:`_`.
```

"interpreted as identifiers" meaning it is a term that consists of a single \@ident?  Would you clarify?

Reply:
Yes, that's the point. I clarified.